SofaKernel/modules/SofaSimulationCore/src/sofa/simulation/MechanicalComputeEnergyVisitor.cpp

```diff
@@ -20,7 +20,9 @@
 * Contact information: contact@sofa-framework.org                             *
 ******************************************************************************/
 #include <sofa/simulation/MechanicalComputeEnergyVisitor.h>
+#include <sofa/core/behavior/ForceField.h>
```

Comment:
It is useful. Some includes have been deleted in BaseMechanicalVisitor.h, because a forward declaration is enough. But you have to include the missing includes in the cpp files.